contrib/prometheus/README.md

```diff
@@ -20,6 +20,7 @@ Add a new job to the main prometheus.conf file:
 ```
 
 ### for Prometheus v2
```

Comment:
(aside: I wonder if we should get rid of the "prometheus v1" section, since it's unmaintained and nobody should be using it)